<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pose Imitate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>400</height>
    </size>
   </property>
   <widget class="QLabel" name="srcImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string>src</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fakeImage">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string>fake</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="poseImage">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>192</width>
      <height>192</height>
     </rect>
    </property>
    <property name="text">
     <string>pose</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>&amp;File(F)</string>
    </property>
    <addaction name="action_open_camera"/>
    <addaction name="action_load_video"/>
    <addaction name="separator"/>
    <addaction name="actione_quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Go(G)</string>
    </property>
    <addaction name="action_openpose"/>
    <addaction name="action_pix2pix"/>
    <addaction name="action_Capture"/>
    <addaction name="action_Run"/>
    <addaction name="action_Stop"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help(H)</string>
    </property>
    <addaction name="action_how_to_use"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open_camera">
   <property name="text">
    <string>&amp;open camera(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_load_video">
   <property name="text">
    <string>&amp;load video(L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actione_quit">
   <property name="text">
    <string>&amp;quit(Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_openpose">
   <property name="text">
    <string>&amp;openpose(O)</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="action_pix2pix">
   <property name="text">
    <string>&amp;pix2pix(P)</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="text">
    <string>&amp;Run(R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="text">
    <string>&amp;Stop(S)</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="action_Capture">
   <property name="text">
    <string>&amp;Capture(C)</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="action_how_to_use">
   <property name="text">
    <string>&amp;how to use(H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;about(A)...</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="action_stop_P">
   <property name="text">
    <string>sto&amp;p(P)</string>
   </property>
  </action>
  <action name="action_start_T">
   <property name="text">
    <string>s&amp;tart(T)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actione_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
